### DISCUSSION TITLE
\\overset{\\leftrightarrow}{O B}

### DISCUSSION THREAD

**Discussion by Kanan02** [ForNeVeR/wpf-math#321] [Q&A] [answered]
Hello, is there anything to convert this expression?
Of there's converter for single arrows it will also suit me)
Thanks in advance for reply

**Comment by Kanan02** [comment]
I found that the problem is in overset what is the replace for it?

**Comment by ForNeVeR** [comment, marked as answer]
The closest I was able to get is `\matrix{\leftrightarrow \\ O B}`:
![image](https://user-images.githubusercontent.com/92793/191094675-c9469e24-b237-460d-96e5-e14340e7e826.png)

**Reply by Kanan02** [reply]
Thanks
